rest/config.go

```diff
@@ -133,6 +133,7 @@ type DbConfig struct {
 	ViewQueryTimeoutSecs *uint32                        `json:"view_query_timeout_secs,omitempty"`     // The view query timeout in seconds
 	LocalDocExpirySecs   *uint32                        `json:"local_doc_expiry_secs,omitempty"`       // The _local doc expiry time in seconds
 	EnableXattrs         *bool                          `json:"enable_shared_bucket_access,omitempty"` // Whether to use extended attributes to store _sync metadata
+	SessionCookieName    *string                        `json:"session_cookie_name"`                   // Custom per-database session cookie name
```

Comment:
This can just be string, not *string.  We don't need to differentiate between session_cookie_name=nil and session_cookie_name="", since "" isn't valid.

Reply:
Fixed